src/doc/python_scripting/functions.rst

```diff
@@ -1660,37 +1639,15 @@ expression : string
     The expression definition as a string.
 
 return type : CLI_return_t
-    The DefineExpression functions return 1 on success and 0 on failure.
+    The DefineMeshExpression function returns 1 on success and 0 on failure.
```

Comment:
Same remarks as for material expressions...not sure we should document it.

Reply:
I started a discussion on this. Perhaps we should provide a way to create a mesh variable.